src/me/corriekay/pokegoutil/utils/Utilities.java

```diff
@@ -70,8 +73,22 @@ public static void sleepRandom(int minMilliseconds, int maxMilliseconds) {
     public static String getRealExceptionMessage(Exception e) {
         String message = e.getMessage();
         if (e instanceof InvalidProtocolBufferException || "Contents of buffer are null".equals(message)) {
-            message = "Server hasn't responded in time. Seems to be busy. The action may have been successful though. (" + message + ")";
+            message = "Server hasn't responded in time. "
+                    + "Seems to be busy. "
+                    + "The action may have been successful though. (" + message + ")";
         }
         return message;
     }
+
+    public static String concatString(char delimeter, String... strings) {
```

Comment:
`return Stream.of(strings).collect(Collectors.joining(String.valueOf(delimiter)));` does the same